test/test_collections.rb

```diff
@@ -132,11 +133,12 @@ class TestCollections < JekyllUnitTest
           _methods/um_hi.md
           _methods/escape-+\ #%20[].md
           _methods/yaml_with_dots.md
-        ], doc.relative_path
+        ), doc.relative_path
       end
     end
 
-    should "not include files which start with an underscore in the base collection directory" do
+    should "not include files which start with an underscore in the base collection "\
+           "directory" do
```

Comment:
Putting just one or two words on the next line are not great... How about:

```ruby
should "not include files from base dir which start with an underscore" do
```

Does that work?